UI/src/assetTypeConfiguration.js

```diff
@@ -50,6 +50,22 @@
       trWeightGroup: [assetType.trWeightLimits, assetType.trTrailerTruckWeightLimits, assetType.trAxleWeightLimits, assetType.trBogieWeightLimits]
     };
 
+    var datePeriodValueExtract = function (date) {
+      var datePeriodValue = date.getPropertyValue().values;
+      var startDate = new Date(_.head(datePeriodValue).value.startDate.replace(/(\d+).(\d+).(\d{4})/, "$2/$1/$3"));
+      var endDate = new Date(_.head(datePeriodValue).value.endDate.replace(/(\d+).(\d+).(\d{4})/, "$2/$1/$3"));
+
+      return [startDate, endDate];
+    };
+
+    var isEndDateAfterStartdate = function (date) {
```

Comment:
I believe this can be optimized to:

```suggestion
    var isEndDateAfterStartdate = function (date) {
		_.reduce(datePeriodValueExtract(date), function(startDate, endDate){ return startDate <= endDate;})
	};
```

Reply:
I choose Miguel solution! 👍